niri-ipc/src/lib.rs

```diff
@@ -167,7 +167,11 @@ pub enum Action {
     /// Open the screenshot UI.
     Screenshot {},
     /// Screenshot the focused screen.
-    ScreenshotScreen {},
+    ScreenshotScreen {
+        /// Save the screenshot in addition to putting it in your clipboard.
+        #[cfg_attr(feature = "clap", arg(short, long, action = clap::ArgAction::Set, default_value_t = true))]
```

Comment:
How does this look on the CLI? Do you have to do `--write-to-disk=false`? Or is there a `--no-write-to-disk`? Could you show what the help looks like?

Reply:
Apparently clap does not support automatic `--no-XYZ` flags (https://github.com/clap-rs/clap/issues/815) so we would have to do it manually. I can implement that if you want. As I said, I have no strong opinion.

Reply:
Eh, I suppose it's fine as is then

Reply:
Bikeshed: should the short flag be -d for disk perhaps rather than -w for write? Or would that be more confusing

Reply:
`-d` also works. I think it's fine to change it later if another flag is added that fits better, especially for the short flag.